notebooks/utils/notebook_utils.py

```diff
@@ -13,18 +13,13 @@
 from pathlib import Path
 from typing import List, NamedTuple, Optional, Tuple
 
-import cv2
```

Comment:
opencv still used in VideoPlayer, please restore import

Reply:
The import is called inside the `VideoPlayer` function

Reply:
but ci logs speak not https://github.com/openvinotoolkit/openvino_notebooks/actions/runs/5267602177/jobs/9523076146?pr=1114

Reply:
I put it in a wrong place 😂
It should work now